test/navigation/AdditionalReferenceNavigatorSpec.scala

```diff
@@ -52,5 +52,47 @@ class AdditionalReferenceNavigatorSpec extends SpecBase with ScalaCheckPropertyC
       }
 
     }
+
+    "in NormalMode" - {
+
+      val mode = NormalMode
+
+      "must go from AdditionalReferenceTypePage to AdditionalReferenceNumberYesNoPage page" in {
+
+        val userAnswers = emptyUserAnswers.setValue(AdditionalReferenceTypePage(additionalReferenceIndex), AdditionalReferenceType("test", "test"))
+
+        navigator
+          .nextPage(AdditionalReferenceTypePage(additionalReferenceIndex), mode, userAnswers)
+          .mustBe(controllers.additionalReference.index.routes.AdditionalReferenceNumberYesNoController.onPageLoad(arrivalId, additionalReferenceIndex, mode))
+      }
+
+      "must go from AdditionalReferenceNumberYesNoPage to AdditionalReferenceNumberPage page" in {
+
+        val userAnswers = emptyUserAnswers.setValue(AdditionalReferenceNumberYesNoPage(additionalReferenceIndex), true)
+
+        navigator
+          .nextPage(AdditionalReferenceNumberYesNoPage(additionalReferenceIndex), mode, userAnswers)
+          .mustBe(controllers.additionalReference.index.routes.AdditionalReferenceNumberController.onPageLoad(arrivalId, additionalReferenceIndex, mode))
+      }
+
+      "must go from AdditionalReferenceNumberYesNoPage to AddAnotherAdditionalReference page when user chooses Yes" in {
```

Comment:
I believe this test description should be for when "No" is selected.

Reply:
Done.